src/test/java/unit/Workspace.java

```diff
@@ -0,0 +1,261 @@
+package unit;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.text.IsEqualIgnoringCase.equalToIgnoringCase;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import bio.terra.cli.serialization.userfacing.UFStatus;
+import bio.terra.cli.serialization.userfacing.UFWorkspace;
+import com.fasterxml.jackson.core.type.TypeReference;
+import harness.TestCommand;
+import harness.TestUsers;
+import harness.baseclasses.ClearContextUnit;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.hamcrest.CoreMatchers;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+/** Tests for the `terra workspace` commands. */
+@Tag("unit")
+public class Workspace extends ClearContextUnit {
+  @Test
+  @DisplayName("status, workspace list reflect workspace create")
```

Comment:
This shows up in the JUnit output, as descriptions of the tests that ran. I find that helpful. If we're worried about consistency, I'd advocate for including this in other repos instead of removing it here.